image-mode/microshift/config/Containerfile.bootc-rhel9

```diff
@@ -0,0 +1,34 @@
+FROM registry.redhat.io/rhel9/rhel-bootc:9.4
+
+ARG USHIFT_VER=4.17
```

Comment:
4.17? it is 4.19 in microshift.sh

Reply:
yes it is just an arg and updated by build.sh script.

Reply:
we'll only start using this from 4.18 so 4.17 is quite unexpected.

Reply:
yes we can update it to 4.18/4.19